Update otel agent dashboard json and add appname subsystem variables

amit-mazor opened the pull request:
Added variables for appname and subsystem and added them to all metrics that have these labels. (queue metrics for instance dont have these labels so no need)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/coralogix/telemetry-shippers?pullRequest=115) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/coralogix/telemetry-shippers?pullRequest=115) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/coralogix/telemetry-shippers?pullRequest=115) it.</sub>

amit-mazor commented:
* About the queue metrics, I was confused between the prometheus and otel specification, fixed now, both in the variables and in the panels themselves. 
* Splitted the panels of the `queues` so we can see clearly the capacity vs current size in each node. 
* Added `$__rate_interval`
* Removed the batch panel for now until we figure out why the values show us small values compared to batch processor configuration

amit-mazor commented:
@povilasv I dont see exporter_failed_logs metrics, maybe because we still dont export logs so this metric doesnt exist , but anyway added the failed_metrics next to the spans, 
and changed the queue panels again a bit

amit-mazor commented:
Pushing since the current json in master doesnt work and is not updated at all, 
If we will want to change something , I'll open a new PR
